linkerd/datadog_checks/linkerd/linkerd.py

```diff
@@ -0,0 +1,43 @@
+# (C) Datadog, Inc. 2018
+# All rights reserved
+# Licensed under a 3-clause BSD style license (see LICENSE)
+
+try:
+    # Agent5 compatibility layer
+    from datadog_checks.errors import CheckException
+    from datadog_checks.checks.prometheus import GenericPrometheusCheck
+except ImportError:
+    from checks import CheckException
+    from checks.prometheus_check import GenericPrometheusCheck
```

Comment:
`GenericPrometheusCheck` won't be in `checks.prometheus_check` in agent5

Check how `kube_proxy` does it: https://github.com/DataDog/integrations-core/blob/master/kube_proxy/datadog_checks/kube_proxy/kube_proxy.py